### DISCUSSION TITLE
Side scroll bar does not appear when trying to select a column to filter with at Provider

### DISCUSSION THREAD

**Discussion by gmentzik** [Budibase/budibase#10002] [Help] [answered]
Hi support team, 

I am in Self host budibase version 2.3.16. and my application is at 2.1.42 (waiting for the version that reverts to previous outline mode before skeleton mode).
I am using Mysql as my data source. I have a provider to a mysql table with many columns, that also have defined relations to other tables. This come up as a big list of columns to select from at provider filter. Budibase backend does not provide me with a side scroll bar thus I cannot select the column I need to filter with. I have tried in both Chrome and Firefox. What could be wrong?

**Comment by melohagan** [comment, marked as answer]
Hey @gmentzik 

Would you mind sharing a screenshot please?

**Reply by gmentzik** [reply]
> Hey @gmentzik
> 
> Would you mind sharing a screenshot please?

![image](https://user-images.githubusercontent.com/65217733/224744925-7d067759-fbe6-457b-9a11-f5b792293375.png)

**Reply by melohagan** [reply]
Thanks @gmentzik I understand now. There was an issue raised for that here: https://github.com/Budibase/budibase/issues/9909

A fix has been implemented, and will be deployed in the next release (if it's not already in the latest version).

**Reply by gmentzik** [reply]
Thank you @melohagan . I will try to check it out.

**Reply by gmentzik** [reply]
update: Updated backend version to 2.4.8 and still scrolling is not there.

**Reply by melohagan** [reply]
It'll be available in the next release then - thank you for your patience!
